docs/usage/user-stories/swissquote.md

```diff
@@ -197,18 +199,42 @@ Here is the dashboard for our current scheduler:
 
 <figure markdown>
   ![Swissquote scheduler dashboard](../assets/images/swissquote_stats.png){ loading=lazy }
-  <figcaption>A dashboard we made at Swissquote to keep our Renovate runs in check, July 2022.</figcaption>
+  <figcaption>A dashboard we made at Swissquote to keep our Renovate runs in check, November 2023.</figcaption>
 </figure>
 
 We don’t force any team to use Renovate, each team can decide to opt-in and do it for each project separately.
 
 Some statistics:
 
-- 824 repositories enabled out of about 2000 active repositories
-- 8000 PRs were merged since we installed Renovate
+- 857 repositories enabled out of about 2000 active repositories
+- 11000 PRs were merged since we installed Renovate
 - 239 PRs were merged last month
 - 2 SSDs died on our Renovate machine with the number of projects to clone again and again
 
+### How does the scheduler work ?
+
+The scheduler is a Node.js application which handles an in-memory queue and starts Docker containers to run Renovate on.
+Our custom scheduler application regularly sends points to an InfluxDB which we then display in Grafana.
+
+Here is how it works:
+
+<figure markdown>
+  ![Swissquote scheduler diagram](../assets/images/swissquote_stats_collection.png){ loading=lazy }
+  <figcaption>The diagram explaining how our scheduler interacts with Renovate.</figcaption>
+</figure>
+
+All the information on the dashboard you saw above is created from three measurements:
+
+1. Queue: Every 5 minutes, we send a status of the queue size and number of jobs currently running
+1. Webhook: When receiving a webhook request, we send a point on the duration of treatment for that item
```

Comment:
```suggestion
1. Webhook: When receiving a webhook request, we send a data point on the duration of treatment for that item
```

It's not clear to me who or what _receives_ a webhook. Is that important to know at this point? Also should it be `point` or `data point`?

Reply:
True, I could change to mention that it's GitHub that sends a webhook